### PR TITLE
Earthely - Business site week3 by Jakob Lindström

### PR DESCRIPTION
**Pull request by jakobxlindstrom**
I had fun and made these things work:
- Hero video that stays as background on scroll
- Form that contains checkbox, radio button and different types of input fields. 
- A accodion with a transition effect and also a change on the + and - symbol. 
- Opacity play making the site feel as if you are floating on top of the globe. 
- A nice imagerow that is easy to modify depending on how many images I need. 
- Adding assets/styling such as google font, images and colors. 

Im proud of my layout and that my form and accordion work accordingly.
Im most proud of getting my transition to finally work in my accordion, it took its time. 

Next time I will start with putting some time into finding fonts, colors and a few assets to make my site feel more alive in the early stages rather than just having placeholders, I have realized that I need that to feel and see the project moving forward. 

Here is my site:

https://earthely.netlify.app

/Jakob

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,30 +1,160 @@
...
+          required />
+
+
+        <label id="form" class="label" for="name">Email Address</label>
+        <input class="name textblock" type="email" id="email" name="email" placeholder="Email" value="" required />
+
+        <br>
+
+        <label id="form" class="label" for="name">Date of birth </label>
+        <input class="name" type="date" id="date" name="date" value="" required />
```
Comment (loulunds) [inline code/index.html:62]: For some reason, I can put 6 digits on the year. Maybe set a min max value? ` min="1900-01-01" max="2021-0101` on input for example.

File: code/style.css [hunk 2]
```diff
@@ -1,7 +1,322 @@
...
+  padding: 1em;
+  margin-bottom: 1em;   
+}
+  
+input:valid{
+      background-color: rgb(76, 205, 76);
+  }
+  
+input:invalid{
+      background-color: rgb(232, 89, 89);
```
Comment (loulunds) [inline code/style.css:271]: The contrast between the text and background color is not so good. I can barely read the text after the background turns red. It's good though that you have some styling to indicate invalidity.

File: code/style.css [hunk 3]
```diff
@@ -1,7 +1,322 @@
...
 
 .active + .answer {
-  /* some styling here */
+  max-height: 5em;
+  font-style: italic;
+  font-size: 18px;
+}
+  
+.active {
+   font-weight: bold;
```
Comment (loulunds) [inline code/style.css:154]: I like this minimal change but causes a big impact on the text. Well done.

File: code/style.css [hunk 4]
```diff
@@ -1,7 +1,322 @@
...
+    justify-content:space-between;
+    flex-wrap: wrap;
+    align-items: center;
+    background-color: #6A8187 ;
+    padding: 5em;
+    color: white;
+    font-size: 20px;
+  }
+  
+@media screen and (max-device-width: 640px) {
```
Comment (loulunds) [inline code/style.css:201]: I see that you didn't start with mobile first. Maybe you might want to try to do a mobile-first approach on your next project?

File: code/style.css [hunk 5]
```diff
@@ -1,7 +1,322 @@
...
+  width: auto;
+  height: auto;
+  z-index: -100;
+  -ms-transform: translateX(-50%) translateY(-50%);
+  -moz-transform: translateX(-50%) translateY(-50%);
+  -webkit-transform: translateX(-50%) translateY(-50%);
+  transform: translateX(-50%) translateY(-50%);
+  /* background: url(placeholder.jpg) no-repeat; */
+  background-size: cover;
+}
```
Comment (loulunds) [inline code/style.css:63]: Love the video!

File: code/style.css [hunk 6]
```diff
@@ -1,7 +1,322 @@
...
+  width: 70%;
+  font-family: 'Work Sans', sans-serif;
+  font-size: 25px;
+  background-color: #B8CABF;
+  border-radius: 0.5em;
+  opacity: 80%;
+}
+
+.accordion:hover{
+  opacity: 100%;
```
Comment (loulunds) [inline code/style.css:144]: I love this too! super smart! Small changes but big impact!

File: code/style.css [hunk 7]
```diff
@@ -1,7 +1,322 @@
...
+
+.header-text{
+  color: whitesmoke;
+  font-family: 'Work Sans', sans-serif;
+  text-align: left;
+  width: 60%;
+  line-height: 1.5em;
+  padding-left: 3em;
+  font-size: 20px;
+}
```
Comment (loulunds) [inline code/style.css:46]: The contrast of the texts are not so good when they are on top of the spinning globe. Maybe put a translucent backgorund just so users can read the text easily.

**Review by loulunds — COMMENTED** [review]
Hej Jakob! Great job with your site! I have some suggestions for improvements otherwise well done! You met all the minimum requirements plus more I guess. I wasn't able to test your website on other physical devices though. I hope that's okay. Looking forward to see more of your work in the future! Happy coding!

File: code/index.html (hunk 1 above)
Comment (jakobxlindstrom) [reply code/index.html:62]: Great observation, I have not even tried filling it manually, I just used the calender that pops-up.

File: code/style.css (hunk 4 above)
Comment (jakobxlindstrom) [reply code/style.css:201]: Great observation, that is my plan for sure! :) Feel comfortable now with the basic code so I will start the other way.

File: code/style.css (hunk 5 above)
Comment (jakobxlindstrom) [reply code/style.css:63]: Me toO!

File: code/style.css (hunk 6 above)
Comment (jakobxlindstrom) [reply code/style.css:144]: Right! I like that you appreciate it.

File: code/style.css (hunk 7 above)
Comment (jakobxlindstrom) [reply code/style.css:46]: I will check out a translucent background, sounds interesting.

**Pull request closed without merging**